.gitlab-ci.yml

```diff
@@ -19,6 +19,15 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-docker-deploy.yml'
 
+
+test:unit:
+ script:
+   - git config --global user.email test@example.com
+   - git config --global user.name  mendertester
```

Comment:
Thinking out loud: wouldn't be cool that the template supports somewhat a "user hook" to run extra shell commands? We have something like that for OS dependencies, where user pipelines can have a `deb-requirements.txt` file that the pipeline installs if existing. So something similar for a user pipeline like this one to run the `git config` commands without needing to copy/paste the rest of the `script` (and risking becoming out of sync).

Anyway, just an idea for the future.